next-sitemap.config.js

```diff
@@ -28,29 +28,37 @@ module.exports = {
 		'sitemap.xml',
 		'/server-sitemap.xml',
 		'/boundary/docs*',
+		'/boundary/tutorials*',
 		'/consul/docs*',
 		'/consul/commands*',
 		'/consul/api-docs*',
+		'/consul/tutorials*',
 		'/nomad/docs*',
 		'/nomad/plugins*',
 		'/nomad/intro*',
 		'/nomad/tools*',
+		'/nomad/tutorials*',
 		'/packer/docs*',
 		'/packer/guides*',
 		'/packer/intro*',
 		'/packer/plugins*',
+		'/packer/tutorials*',
 		'/terraform/docs*',
 		'/terraform/cli*',
 		'/terraform/internals*',
 		'/terraform/intro*',
 		'/terraform/language*',
+		'/terraform/tutorials*',
 		'/vagrant/docs*',
 		'/vagrant/intro*',
 		'/vagrant/vagrant-cloud*',
+		'/vagrant/tutorials*',
 		'/waypoint/docs*',
 		'/waypoint/commands*',
 		'/waypoint/plugins*',
+		'/waypoint/tutorials*',
 		'/vault/docs*',
 		'/vault/api-docs*',
+		'/vault/tutorials*',
```

Comment:
We're missing 'well-architected-framework/*' here. [Whimsical sitemap ](https://whimsical.com/dark-mode-research-dev-dot-sitemap-SqDL3eKRmwmW3cMpAC7EUr)for reference.

Reply:
https://developer.hashicorp.com/well-architected-framework

Reply:
Good catch!